src/shared/services/api/api.js

```diff
@@ -37,6 +37,7 @@ export function* authCall(url, params, authorizationToken) {
 
 export function* authCallWithPayload(url, params, method) {
   const headers = {
+    accept: 'application/json',
```

Comment:
This will most likely be corrected, but maybe this should be `Accept` for consistency

Reply:
Headers are case-**in**sensitive. See https://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html#sec4.2 for reference.